Adafruit_SSD1306.cpp

```diff
@@ -592,7 +592,7 @@ boolean Adafruit_SSD1306::begin(uint8_t vcs, uint8_t addr, boolean reset,
     @param  y
             Row of display -- 0 at top to (screen height -1) at bottom.
     @param  color
-            Pixel color, one of: BLACK, WHITE or INVERT.
+            Pixel color, one of: SSD1306_BLACK, _WHITE or _INVERT.
```

Comment:
The _WHITE and _INVERT should be updated to match the new names SSD1306_WHITE and SSD1306_WHITE. I'm guessing this was just a typo.